website/content/en/preview/getting-started-with-terraform/_index.md

```diff
@@ -42,19 +42,25 @@ After setting up the tools, set the following environment variables to store
 commonly used values.
 
 ```bash
-export CLUSTER_NAME="${USER}-karpenter-demo"
+export CLUSTER_NAME="karpenter-ex"
```

Comment:
well we could remove it since the cluster name is set as a Terraform local variable so users are only concerned with `terraform init` and `terraform apply`, but the two bespoke manifest files at the bottom of the example still reference `${CLUSTER_NAME}`. So the change here is really to sync it back up with what is set in the local variable.